### PR TITLE
UIOR-694 Add a setting for Save and Add next POL

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
https://issues.folio.org/browse/UIOR-694

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/common/hooks/useLinesLimit/useLinesLimit.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+export const useLinesLimit = (enabled = true) => {
+  const ky = useOkapiKy();
+
+  const searchParams = {
+    query: `(module=${MODULE_ORDERS} and configName=${CONFIG_LINES_LIMIT})`,
+  };
+
+  const { isLoading, data = {} } = useQuery(
+    [useNamespace, 'order-lines-limit'],
```
Comment (axelhunn) [inline src/common/hooks/useLinesLimit/useLinesLimit.js:23]: Does it work this way?
I mean `useNamespace`

File: src/common/hooks/useLinesLimit/useLinesLimit.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```
Comment (axelhunn) [inline src/common/hooks/useLinesLimit/useLinesLimit.test.js:1]: This is not needed anymore

**Review by axelhunn — APPROVED** [review]

**Comment by NikitaSedyx** [comment]
I think it's not the best approach to clear a lot in submit (additionally form is pretty complex and it can produce issues), what about controlling flag by route state?

**Comment by AndreiShumski2** [comment]
> I think it's not the best approach to clear a lot in submit (additionally form is pretty complex and it can produce issues), what about controlling flag by route state?

Not sure I got your idea, but the route doesn't change after submit if `create another` is checked. Let's find the best approach :)

**Review by NikitaSedyx — APPROVED** [review]

File: src/components/LayerCollection/LayerPOLine.js [hunk 3]
```diff
@@ -186,10 +189,18 @@ function LayerPOLine({
...
+        : `/orders/view/${id}/po-line/view/${savedLine.id}`;
+      const state = isCreateAnotherChecked ? { isCreateAnotherChecked: true } : {};
+
       history.push({
-        pathname: `/orders/view/${id}/po-line/view/${savedLine.id}`,
+        pathname,
         search,
+        state,
       });
+      setSavingValues();
```
Comment (NikitaSedyx) [inline src/components/LayerCollection/LayerPOLine.js:202]: is it still required?

Comment (AndreiShumski2) [reply src/components/LayerCollection/LayerPOLine.js:202]: yep. otherwise the form will not be cleared after saving

**Comment by AndreiShumski2** [comment]
@dltj Could you please update translations? Thanks

**Comment by dltj** [comment]
https://github.com/folio-org/ui-orders/pull/1155

**Pull request merged**